examples/forcefield_modification/README.md

```diff
@@ -1,3 +1,3 @@
-## Manipulating SMIRNOFF parameters after loading into a `ForceField` object
+## `forcefield_modification`: Manipulating SMIRNOFF parameters after loading into a `ForceField` object
```

Comment:
We don't need to include the name of the directory in any of these `README`s.

Reply:
On the dedicated Examples page, all of the individual example READMEs are imported using `mdinclude`. This was leading to clashing header styles, and since some of the READMEs have manifests, code blocks, and sub-headers, I was finding it really hard to differentiate one example from another. After trying a few things, I settled on the inline-code style in the headers to make it easier to separate one example from another. 

This is a short-term solution and I'm happy to remove it.

Reply:
How about this:
* Since we've migrated code out of the `README`s, let's _not_ `mdinclude` the README contents and just link to the relevant github directories.
* Once we finish converting the examples to Jupyter notebooks, we can add [mybinder](https://mybinder.org/) or [colab](https://colab.research.google.com) links to allow users to jump right into the example notebooks via the web.

Reply:
Update: After looking at this for a bit longer, I also don't like the new style. Reverted in eaaf7db.

Reply:
Ah, I'd missed your previous comment. I like your idea -- Removed the individual example mdincludes in e2d76b4 , and added links to `master` branch location of the examples.